src/bot.py

```diff
@@ -9,7 +9,7 @@
 import sys
 from aiohttp import web
 from accounting import LedgerServer, Authorization, RedditAccountId, DiscordAccountId, AccountId
-from commands import COMMANDS, list_commands_as_markdown, CommandException, assert_authorized, process_command
```

Comment:
I do hope none of these show up in this file as they really shouldn't because otherwise you might be in trouble with running this